src/scene/tools/vcSceneTool.h

```diff
@@ -12,12 +12,14 @@ class vcSceneTool
   virtual void SceneUI(vcState *pProgramState);
   virtual void HandlePicking(vcState *pProgramState, vcRenderData &renderData, const vcRenderPickResult &pickResult);
   virtual void PreviewPicking(vcState *pProgramState, vcRenderData &renderData, const vcRenderPickResult &pickResult);
+  virtual void Cancel(vcState *pProgramState); // Essentially a 'cancel' command has been issued (esc key was pressed)
```

Comment:
This comment seems like it's trying to avoid confusion since "Cancel" could be interpreted as either an action or a reaction, and in this case we're wanting a reaction. Assuming this is correct, perhaps calling this function `OnCancel` would be better? This is generally how UI naming works, for example, a button has a `Click` and `OnClick` function.

Reply:
`OnCancel` is more consistent- `OnNodeUpdate` etc